src/assets/Signin.css

```diff
@@ -0,0 +1,52 @@
+.container{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    width: 100%;
+    height: 100%;
+
+}
+
+.inputContainer{
+    display: flex;
+    flex-direction: column;
+
+    gap: 30px;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+}
+
+.title{
+    color: #854D27;
```

Comment:
Avoid using hard coded colors, Maybe create a variables.css file for now. 
However we will move towards having a standard theme template, lets discuss this in the next meeting as well.